examples/projects/bsoncxx/cmake/static/build.sh

```diff
@@ -9,7 +9,7 @@ CMAKE=${cmake_binary:-cmake}
 rm -rf build/*
 cd build
 if [ -z "$MSVC" ]; then
-    "$CMAKE" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" -DCMAKE_CXX_STANDARD="${CXX_STANDARD}" ..
+    "$CMAKE" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" -DCMAKE_CXX_STANDARD="${CXX_STANDARD}" -DCMAKE_INSTALL_PREFIX="../../../../../../build/install" ..
```

Comment:
Revert? What motivated this change?

Reply:
It appears this is no longer needed. I believe this was needed when static linking the C++ driver with libmongoc but the tests still needed to link with libmongoc, however since static linking has been reverted, this became unecessary.